CMakeLists.txt

```diff
@@ -100,10 +106,51 @@ endif()
 
 # Install
 if(GLAD_INSTALL)
+
+  set(config_install_dir "lib/cmake/glad")
+  set(version_config "${CMAKE_CURRENT_BINARY_DIR}/gladConfigVersion.cmake")
+  set(project_config "${CMAKE_CURRENT_BINARY_DIR}/gladConfig.cmake")
+  set(targets_export_name "gladTargets")
+  set(namespace "glad::")
```

Comment:
I've added a bunch of variables here to make editing paths and other install related properties easier.